lib/matplotlib/tests/test_lines.py

```diff
@@ -304,6 +304,17 @@ def test_marker_as_markerstyle():
     assert_array_equal(line3.get_marker().vertices, triangle1.vertices)
 
 
+@image_comparison(['striped_line.png'], remove_text=True, style='mpl20')
```

Comment:
Interestingly, `remove_text` does not apply to legend. (Should be investigated at some point).

Still, I propose not to pick up text in the reference image. Do you want the legend entries as a test that the lines are drawn correctly there? If not remove the legend, if yes use `label=' '`.

Reply:
@oscargus asked for the legend so I guess he wanted to check the legend lines are drawn correctly.

Reply:
Correct! Completely happy without text!